docs/content/users/extend/custom-commands.md

```diff
@@ -65,6 +65,24 @@ Changes to the command files in the global `.ddev` directory need a `ddev start`
 
 There are many examples of [global](https://github.com/ddev/ddev/tree/master/pkg/ddevapp/global_dotddev_assets/commands) and [project-level](https://github.com/ddev/ddev/tree/master/pkg/ddevapp/dotddev_assets/commands) custom/shell commands that ship with DDEV you can adapt for your own use. They can be found in your `~/.ddev/commands/*` directories and in your project’s `.ddev/commands/*` directories. There you’ll see how to provide usage, examples, and how to use arguments provided to the commands. For example, the [`xdebug` command](https://github.com/ddev/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/web/xdebug) shows simple argument processing and the [launch command](https://github.com/ddev/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/host/launch) demonstrates flag processing.
 
+## Command Line Completion
+
+If your custom command has a set of pre-determined valid arguments it can accept, you can use the [`AutocompleteTerms`](#autocompleteterms-annotation).
+
+For dynamic completion, you can create a separate script with the same name in a directory named `autocomplete`.
+For example, if your command is in `~/.ddev/commands/web/my-command`, your autocompletion script will be in `~/.ddev/commands/web/autocomplete/my-command`.
```

Comment:
Probably needs link to real example. You probably removed the real example link because it was breaking link check :) You can exclude a link check by adding to the markdown-link-check.json.

Reply:
https://github.com/ddev/ddev/pull/5785#issuecomment-1934236761
> it probably doesn't need a link in docs

Just to clarify, since you're saying the opposite now... You *do* want the link now?

Reply:
Yes, I think it needs real, maintained example. Just exclude it from the check or whatever. I didn't realize before out of context how important a real maintained example was.

Reply:
Sweet, added the link back.

Reply:
> You can exclude a link check by adding to the markdown-link-check.json.

I don't think we should exclude this link there just to pass CI in the PR, because then as soon as the PR gets merged we'd need a separate PR just to remove it from the file, so that the link _can_ be checked from then onwards (in case the file gets deleted or renamed in the future, we still _do_ want to check the link!)